### ISSUE TITLE
Cannot build

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#33]
```
What steps will reproduce the problem?
build.bat compile 

What is the expected output? What do you see instead?
Package com.sun.javadoc doesn't exists.

What version of the product are you using? On what operating system, file
system, and virtual machine?

Windows XP SP3
JDK 1.6.0_06
H2 1.0.76

My JAVA_HOME is C:\Program files\Java\jdk1.6.0_06

I could fix this problem by this.
copy "%JAVA_HOME%\lib\*" "%JAVA_HOME%\..\lib\" 

```

Original issue reported on code.google.com by `yukoba` on 12 Aug 2008 at 5:39

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I have a very similar system (Windows XP, JDK 1.6) and it works for me. Could 
you
post the output for:

echo %JAVA_HOME%
echo %CLASSPATH%
dir "%JAVA_HOME%\lib\tools.jar"
java -version
javah -version
javac
javadoc
javac -source 1.6
javac -source 1.5
javac -source 1.4

The last three is only to find out if javac says "javac: invalid source 
release: 1.5"
or "javac: no source files".

Regards,
Thomas

```

Original comment by `thomas.t...@gmail.com` on 13 Aug 2008 at 1:41

**Comment by GoogleCodeExporter** [comment]
```
I'm using Japanese Windows XP, and error messages are Japanese, so I translate 
them
to English.

C:\Program Files\H2>echo %JAVA_HOME%
C:\Program Files\Java\jdk1.6.0_06

C:\Program Files\H2>echo %CLASSPATH%
.;C:\Program Files\Java\jre1.6.0_06\lib\ext\QTJava.zip

C:\Program Files\H2>dir "%JAVA_HOME%\lib\tools.jar"

 C:\Program Files\Java\jdk1.6.0_06\lib 

2008/03/25  04:27        12,495,351 tools.jar

C:\Program Files\H2>java -version
java version "1.6.0_06"
Java(TM) SE Runtime Environment (build 1.6.0_06-b02)
Java HotSpot(TM) Client VM (build 10.0-b22, mixed mode, sharing)

C:\Program Files\H2>javah -version
javah version "1.6.0_06"

C:\Program Files\H2>javac
Usage: javac <options> <source files>

C:\Program Files\H2>javadoc
javadoc: Error - Package or class is not specified. <- This was Japanese.
.. Skip ..

C:\Program Files\H2>javac -source 1.6
javac: no source files.
Usage: javac <options> <source files>

C:\Program Files\H2>javac -source 1.5
javac: no source files.
Usage: javac <options> <source files>

C:\Program Files\H2>javac -source 1.4
javac: no source files.
Usage: javac <options> <source files>

```

Original comment by `yukoba` on 13 Aug 2008 at 11:41

**Comment by GoogleCodeExporter** [comment]
```
I found the cause and the solution.

I added
  System.out.println(System.getProperty("java.home")); 
to src\tools\org\h2\build\Build.java compile() and it says
  C:\Program Files\Java\jre1.6.0_06

Because JDK 1.6.0_06 installer copys JRE's java.exe to 
C:\windows\system32\java.exe,
and line 7 of build.bat java.exe points C:\windows\system32\java.exe, and this 
is
JRE's java.exe.

build.bat works properly after I changed line 7 from java to 
"%JAVA_HOME%\bin\java".

```

Original comment by `yukoba` on 13 Aug 2008 at 3:58

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks a lot for your help! I will change build.bat and build.sh in the next 
release
accordingly.

Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 14 Aug 2008 at 1:30
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.77 (2008-08-16)
```

Original comment by `thomas.t...@gmail.com` on 16 Aug 2008 at 7:47
- Changed state: **Fixed**
